datastream-kafka/src/main/java/com/linkedin/datastream/kafka/KafkaProducerWrapper.java

```diff
@@ -98,6 +104,13 @@
   private final DynamicMetricsManager _dynamicMetricsManager;
   private final String _metricsNamesPrefix;
 
+  // A lock used to synchronized access to operations performed on the _kafkaProducer object
```

Comment:
nit: A lock used to "synchronize" access to

Reply:
done